packages/math-input/src/components/keypad-context.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * KeypadContext provides a way to the Keypad and (Server)ItemRenderer to
+ * communicate.
+ *
+ * The KeypadContext.Provider wraps the ExerciseFooter while KeypadContext.Consumer
+ * wraps each (Server)ItemRenderer render site and the Keypad rendered in the
+ * ExerciseFooter.
```

Comment:
**question**: Is this doc comment still accurate?  I feel like we should talk about the `<StatefulKeypadContextProvider />` at least, right?